tests/connection/broker.cpp

```diff
@@ -52,7 +56,11 @@ class Subscriber
         for( size_t i = 0; i < 100 && !received ; ++i )
         {
             if( subscriber.receive( 100 ))
-                received = true;
+            {
+#ifdef ZEQ_USE_ZEROBUF
+                received = echo.gotData && received;
+#endif
+            }
         }
     }
```

Comment:
Shouldn't this be two separate tests? Right now the FB code path is not tests as far as I understand.